### PR TITLE
feat: add support for Radiomaster Pocket radio

### PR DESCRIPTION
**Pull request by 3djc**
Support for: https://www.radiomasterrc.com/products/pocket-radio-controller-m2

https://youtu.be/gOchQwqCIxo

### CODE REVIEW

File: radio/src/targets/taranis/hal.h [hunk 1]
```diff
@@ -819,12 +819,7 @@
   #define STORAGE_SWITCH_E
   #define HARDWARE_SWITCH_E
   #define SWITCHES_GPIO_REG_E           GPIOG
-  #define SWITCHES_GPIO_PIN_E           LL_GPIO_PIN_10  // PG.10
-#elif defined(RADIO_T20)
```
Comment (raphaelcoeffic) [inline radio/src/targets/taranis/hal.h:823]: This looks fishy.

File: radio/src/targets/taranis/hal.h [hunk 2]
```diff
@@ -874,12 +869,7 @@
   #define SWITCHES_GPIO_REG_F_L         GPIOD
   #define SWITCHES_GPIO_PIN_F_L         LL_GPIO_PIN_14 // PD.14
   #define SWITCHES_GPIO_REG_F_H         GPIOE
-  #define SWITCHES_GPIO_PIN_F_H         LL_GPIO_PIN_1  // PE.01
-#elif defined(RADIO_TPROV2)
```
Comment (raphaelcoeffic) [inline radio/src/targets/taranis/hal.h:878]: This as well.

**Comment by pfeerick** [comment]
Yeah, looks like merging main rather than rebasing on main removed the T20 and TProV2 🤭

**Comment by rotorman** [comment]
Default contrast of 10 instead of 15 works best on my Pocket.

**Comment by pfeerick** [comment]
Same here (contrast is better at 10 than 15), but let's do that later once we have more samples as this can vary from display to display somewhat.

**Pull request merged**